scala-package/pom.xml

```diff
@@ -332,9 +332,9 @@
       <version>1.10</version>
     </dependency>
     <dependency>
-      <groupId>log4j</groupId>
-      <artifactId>log4j</artifactId>
-      <version>1.2.17</version>
+      <groupId>org.apache.logging.log4j</groupId>
```

Comment:
we use slf4j in some places, is this used somewhere, can we use consistently use slf4j?

Reply:
@nswamy https://mvnrepository.com/artifact/org.slf4j/slf4j-log4j12/1.7.25 checked here and it seemed compatible and we can see the output logs...